deploy-prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+cd /var/www/dds
+
+# Update mes-aides
+git fetch origin
+git rebase origin/master
```

Comment:
No, that's not what we want.
We want the deployment to be bullet-proof, and git will complain and stop if, for example, some random file is created (repo is not clean), or if there was a force push at some point (exactly what you can do under stress, which would then break the CD, leading to more stress and more harmful behaviour).

You need to [force a pure remote mirror](https://github.com/sgmap/paie-api/blob/master/deploy.sh#L19-L23).